src/components/typeahead/typeahead.component.ts

```diff
@@ -99,13 +119,17 @@ export class TypeaheadComponent<T = any> implements OnChanges, OnDestroy {
     /** Emit the highlighted element when it changes */
     @Output() highlightedElementChange = new EventEmitter<HTMLElement>();
 
+    /** Emits when recently selected options change.*/
+    @Output() recentOptionsChange =  new EventEmitter<T[]>();
+
     activeKey: string = null;
     clicking = false;
     hasBeenOpened = false;
     highlighted$ = new BehaviorSubject<TypeaheadVisibleOption<T>>(null);
     highlightedKey: string = null;
     loadOptionsCallback: InfiniteScrollLoadFunction;
     visibleOptions$ = new BehaviorSubject<TypeaheadVisibleOption<T>[]>([]);
+    recentOptionsWrapped$ = new BehaviorSubject<TypeaheadVisibleOption<T>[]>(null);

```

Comment:
Would it be safer here to default to a value of `[]` instead of null?

Reply:
The value null means that the feature is not used. The empty list means we want to have the history entries, but at the moment, there is no history yet.